webhook_features/src/features/summary_comment.rs

```diff
@@ -230,7 +250,7 @@ async fn refresh_summary_comment(ctx: &Context, repo: Repository, pr_number: u64
 
     let pr_author = pr.user.unwrap().login;
     for comment in all_comments.into_iter() {
-        if comment.user == pr_author {
+        if comment.user == pr_author || ignore_users.contains(&comment.user) {
```

Comment:
What about keeping the review and grouping it as "ignored review"?

Reply:
Good idea, will do.

Reply:
Done